code/datums/abilities/critter/small_animal.dm

```diff
@@ -204,6 +204,93 @@
 			MT.TakeDamageAccountArmor("All", rand(2,8), 0, 0, DAMAGE_STAB)
 		return 0
 
+/datum/targetable/critter/scorpion_sting
```

Comment:
Why make a whole new ability for this instead of subtyping `/datum/targetable/critter/sting`? Seems identical other than the sound and the different amounts for each poison.